packages/node-runtime-worker-thread/package.json

```diff
@@ -52,6 +52,7 @@
   },
   "dependencies": {
     "interruptor": "^1.0.1",
-    "system-ca": "^2.0.1"
+    "system-ca": "^2.0.1",
+    "web-worker": "^1.3.0"
```

Comment:
Hm ... maybe we actually need to create a separate SBOM for this package: https://jira.mongodb.org/browse/MONGOSH-1856

Not something for this PR, but a good reminder